Core/src/org/sleuthkit/autopsy/centralrepository/persona/PersonaDetailsDialog.java

```diff
@@ -44,6 +44,10 @@
         "PersonaDetailsDialogEditTitle=Edit Persona",
         "PersonaDetailsDialogViewTitle=View Persona",})
     public PersonaDetailsDialog(Component parent, PersonaDetailsMode mode, Persona persona, PersonaDetailsDialogCallback callback) {
+        // by default, display the dialog right away
+        this(parent, mode, persona, callback, true);
+    }
+    public PersonaDetailsDialog(Component parent, PersonaDetailsMode mode, Persona persona, PersonaDetailsDialogCallback callback, boolean displayDialog) {
```

Comment:
Instead of passing in whether or not to display the dialog I would just have the calling code call display.  This will also help fix the lightbulb warnings in netbeans that you are not using your new instances.

Reply:
This dialog is created from several places.  (including persona manager)  Everywhere else its OK to immediately display the dialog.  So, far this is the first place where I needed to delay the display in order to pre-populate it.  Forcing every client to explicitly call display seems excessive.